src/lib/core/datetime/date-selection.ts

```diff
@@ -0,0 +1,150 @@
+/**
```

Comment:
why does this file show as new?

Reply:
I have no idea 😕 Some GIT merging issue?? Should I look into it or do we not bother as it will all be merged into one commit eventually.